src/main/kotlin/uk/gov/dluhc/notificationsapi/config/NotifyClientConfiguration.kt

```diff
@@ -215,14 +269,25 @@ class VoterCardNotifyEmailTemplateConfiguration(
     ninoNotMatchedWelsh = null,
     ninoNotMatchedRestrictedDocumentsListEnglish = null,
     ninoNotMatchedRestrictedDocumentsListWelsh = null,
+    rejectedParentGuardianEnglish = null,
+    rejectedParentGuardianWelsh = null,
+    rejectedQualifyingAddressEnglish = null,
+    rejectedQualifyingAddressWelsh = null
 )
 
-abstract class AbstractNotifyEmailTemplateConfiguration(
+data class NotifyLetterTemplateConfiguration(
+    val voterCard: VoterCardNotifyLetterTemplateConfiguration,
+    val postal: PostalNotifyLetterTemplateConfiguration,
+    val proxy: ProxyNotifyLetterTemplateConfiguration,
+    val overseas: OverseasNotifyLetterTemplateConfiguration
+)
+
+abstract class AbstractNotifyLetterTemplateConfiguration(
     val sourceType: SourceType,
     val receivedEnglish: String?,
     val receivedWelsh: String?,
-    val approvedEnglish: String?,
```

Comment:
Is there a reason why the email template has "approved" still but the letter one has "rejected"?

Reply:
Not sure. These templates weren't made by me. This code change was moving the abstract classes to the top of the classes that would inherit from them, just to make things clearer for future readers